op-batcher/batcher/driver.go

```diff
@@ -271,10 +294,21 @@ func (l *BatchSubmitter) loop() {
 	// start the receipt/result processing loop
 	receiptLoopDone := make(chan struct{})
 	defer close(receiptLoopDone) // shut down receipt loop
+
+	var txpoolState atomic.Int32
+	txpoolState.Store(TxpoolGood)
 	go func() {
 		for {
 			select {
 			case r := <-receiptsCh:
+				if errors.Is(r.Err, txpool.ErrAlreadyReserved) && txpoolState.CompareAndSwap(TxpoolGood, TxpoolBlocked) {
+					l.Log.Info("incompatible tx in txpool")
+				} else if r.ID[0].chID == cancelID &&
```

Comment:
...and could here just test
```suggestion
				} else if r.isCancel &&
```

Reply:
done